src/lib/gauge/Gauge.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+	import { actions, type UseActions } from '../actions';
+	import { twMerge } from 'tailwind-merge';
+
+	export let className: string | undefined = undefined;
+	export let use: UseActions = [];
+	export let percentage: number = 0;
+	export let leftLabel: string = '';
+	export let rightLabel: string = '';
+
+	const max = 100;
+	const min = 0;
+	const range = max - min;
```

Comment:
would it be useful to allow setting these as props and allow non percentage based guages?